src/desktop/apps/collect/server.tsx

```diff
@@ -48,6 +50,7 @@ const index = async (req, res, next) => {
       },
     })
 
+    cacheHtmlForPage({ status, key: req.url, html: layout })
```

Comment:
So this isn't real middleware? Seems weird, too, that this is mutating something in the background rather than passing in the url and returning the cached layout, to be passed to `res.send(layout)` below.

Reply:
Yea, I hear you. Will try to refactor this middleware.